test/testutil.py

```diff
@@ -147,3 +149,270 @@ def create_filesystem_customizations(rootfs: str):
+        self._lookaside_conf = conf
+
+
+def ensure_registry():
+    registry_container_name = subprocess.run([
+        "podman", "ps", "-a", "--filter", "name=registry", "--format", "{{.Names}}"
+    ], check=True, capture_output=True).stdout.decode("utf-8").strip()
+
+    if registry_container_name != "registry":
+        subprocess.run([
+            "podman", "run", "-d",
+            "-p", "5000:5000",
+            "--restart", "always",
+            "--name", "registry",
+            "registry:2"
+        ], check=True, capture_output=True)
+
+    registry_container_state = subprocess.run([
+        "podman", "ps", "-a", "--filter", "name=registry", "--format", "{{.State}}"
+    ], check=True, capture_output=True).stdout.decode("utf-8").strip()
+
+    if registry_container_state in ("paused", "exited"):
+        subprocess.run([
+            "podman", "start", "registry"
+        ], check=True, capture_output=True)
+
+
+def get_signed_container_ref(local_registry: str, container_ref: str):
+    container_ref_path = container_ref[container_ref.index('/'):]
+    return f"{local_registry}{container_ref_path}"
+
+
+def sign_container_image(gpg_conf: GPGConf, registry_conf: RegistryConf, container_ref):
+    gpg_gen_key(gpg_conf)
+    ensure_registry()
+    local_registry = registry_conf.local_registry
+    policy_file = registry_conf.policy_file
+    lookaside_conf_file = registry_conf.lookaside_conf_file
+    lookaside_conf = registry_conf.lookaside_conf
+    pub_key_file = gpg_conf.pub_key_file
+    registry_policy = {
+        "default": [{"type": "insecureAcceptAnything"}],
+        "transports": {
+            "docker": {
+                f"{local_registry}": [
+                    {
+                        "type": "signedBy",
+                        "keyType": "GPGKeys",
+                        "keyPath": f"{pub_key_file}"
+                    }
+                ]
+            },
+            "docker-daemon": {
+                "": [{"type": "insecureAcceptAnything"}]
+            }
+        }
+    }
+    with open(policy_file, mode="w", encoding="utf-8") as f:
+        f.write(json.dumps(registry_policy))
+
+    with open(lookaside_conf_file, mode="w", encoding="utf-8") as f:
+        f.write(lookaside_conf)
+
+    signed_container_ref = get_signed_container_ref(local_registry, container_ref)
+    system_lookaside_conf_file = os.path.join(
+        "/etc/containers/registries.d",
+        os.path.basename(lookaside_conf_file)
+    )
+    # We need to temporarily configure system's lookaside for skopeo to
+    # create the sigstore dir.
+    shutil.copy(lookaside_conf_file, system_lookaside_conf_file)
+    subprocess.run([
+        "skopeo", "copy",
+        "--dest-tls-verify=false",
+        "--remove-signatures",
+        "--sign-by", gpg_conf.email,
+        f"docker://{container_ref}",
+        f"docker://{signed_container_ref}",
+    ], check=True, capture_output=True, env={"GNUPGHOME": gpg_conf.home_dir})
```

Comment:
Same question about `capture_output=True` as above about it potentially eating useful debug output (but feel free to just ignore if the answer is the same as for the other question above :)